contributing.md

```diff
@@ -61,7 +63,7 @@ Before starting major work, you should touch base with the maintainers of Featur
 
      **Ubuntu**:
      ```console
-     sudo apt install openjdk-11-jre openjdk-11-jdk scala graphviz -y
+     sudo apt install openjdk-11-jre openjdk-11-jdk scala pandoc -y
```

Comment:
Are the graphviz requirements different for Ubuntu than they are for macOS? We removed graphviz here, but not for the macOS instructions above.

Reply:
This was a mistake. Fixed